src/views/project-board/project-bcf/ProjectBcf.vue

```diff
@@ -539,7 +582,26 @@ export default {
         pushNotification({
           type: "error",
           title: t("Error"),
-          message: t("ProjectBcf.importBcfNotificationError")
+          message: t("ProjectBcf.exportBcfNotificationError")
+        });
+      }
+    };
+
+    const deleteBcfTopics = async () => {
+      try {
+        await selectedTopics.value.forEach(topic => {
```

Comment:
Maybe a `Promise.all` would be more efficient here.

Reply:
`Promise.all` is mandatory here ! `await` of forEach does not await all iterations, it awaits the return of the forEach method which is `undefined`.

Reply:
Maybe a `async.eachLimit` sending requests 10 by 10 to avoid overload in edge cases

Reply:
I think an even better solution would be to change your `deleteTopic` method in `src/state/bcf.js` into `deleteTopics` and make accept a list of topics as a parameters (instead of just a single one).
You can then use `async.eachLimit` in that method to trigger multiple delete requests in parallel.
And once finished call `await loadBcfTopics()` to update the state.
Let me know if you need some more explanation about that. :)